x-pack/plugins/cloud_security_posture/public/components/cloud_security_data_table/use_styles.ts

```diff
@@ -27,18 +27,20 @@ export const useStyles = () => {
     & .euiDataGrid--headerUnderline .euiDataGridHeaderCell {
       border-bottom: ${euiTheme.border.width.thick} solid ${euiTheme.colors.fullShade};
     }
-    & .euiDataGridRowCell__contentByHeight + .euiDataGridRowCell__expandActions {
-      padding: 0;
-    }
     & .euiButtonIcon[data-test-subj='docTableExpandToggleColumn'] {
       color: ${euiTheme.colors.primary};
     }
 
     & .euiDataGridRowCell {
       font-size: ${euiTheme.size.m};
-    }
-    & .euiDataGridRowCell__expandFlex {
+      // Vertically center content
+      display: flex;
       align-items: center;
```

Comment:
@elastic/cloud-security-posture I was unable to figure out how to get vulnerabilities data set up locally to test/QA this datagrid, so I'm not sure if the changes here work as expected without regressions. I would really appreciate your team's help QAing this to ensure this looks correct to you.